package.json

```diff
@@ -32,6 +32,7 @@
   "devDependencies": {
     "chai": "^3.2.0",
     "coveralls": "^2.11.2",
+    "graceful-fs": "4.1.4",
```

Comment:
How persistent were the EMFILE errors you'd observed? Not something I tended to struggle with before, and I'd prefer to drop this unless it's really necessary.